client/src/components/TaskListView/TaskListView.js

```diff
@@ -4,19 +4,33 @@ import TaskCard from "../TaskCard/TaskCard";
 import { PropTypes } from "prop-types";
 
 const TaskListView = ({ tasks, edittable }) => {
-  const cards = tasks.map((task, ix) => (
-    <li key={ix} style={{ listStyle: "none" }}>
-      <TaskCard
-        title={task.title}
-        location={task.location}
-        startTime={task.startTime}
-        endTime={task.endTime}
-        isOngoing={task.isOngoing}
-        showDetailsDialog={() => {}}
-        edittable={edittable}
-      />
-    </li>
-  ));
+  const getDateTime = (taskDateISOString) => {
```

Comment:
This actual function can be moved to the TaskCard component. Since it's the TaskCard that's in charge of the presentation of how the date should look like

Reply:
Moved `getDateTime` to `TaskCard`.